tests/benches/bench_recovery.rs

```diff
@@ -131,20 +135,42 @@ fn bench_recovery(c: &mut Criterion) {
             },
         ),
         (
-            "small-batch(1KB)".to_owned(),
+            "small-batch-format-V1(1KB)".to_owned(),
             Config {
                 region_count: 100,
                 batch_size: ReadableSize::kb(1),
                 item_size: ReadableSize(256),
                 entry_size: ReadableSize(32),
+                format_version: 1,
                 ..Default::default()
             },
         ),
         (
-            "10GB".to_owned(),
+            "small-batch-format-V2(1KB)".to_owned(),
```

Comment:
No need to keep all versions available. During transition (change of default version), you can provide a benchmark report that shows the diff of two commits.

Reply:
Got it.